specs/sifive.com/SCIE/decoder/0.1.0/SCIEDecoder_rtl.json5

```diff
@@ -0,0 +1,37 @@
+{
+  abstractionDefinition: {
+    vendor: 'sifive.com',
+    library: 'core',
+    name: 'SCIEDecoder_rtl',
+    version: '0.1.0',
+    busType: {
+      vendor: 'sifive.com',
+      library: 'core',
+      name: 'SCIEDecoder',
+      version: '0.1.0',
+    },
+    ports: {
+      insn: {
+        description: 'instruction word',
+        wire: {
+          onMaster: {width: 32, direction: 'out', presence: 'required'},
+          onSlave:  {width: 32, direction: 'in', presence: 'required'}
+        }
+      },
+      unpipelined: {
+        description: 'unpipelined unit to be used',
+        wire: {
+          onMaster: {width: 1, direction: 'out', presence: 'required'},
+          onSlave:  {width: 1, direction: 'in', presence: 'required'}
```

Comment:
Are the directions for both this and the `pipelined` port flipped? The slave should have this as an output signal, correct?

Reply:
fixed